docs/self-managed/platform-deployment/helm-kubernetes/deploy.md

```diff
@@ -137,6 +137,27 @@ helm install <RELEASE_NAME> camunda/camunda-platform --version 8.1 \
     --values https://raw.githubusercontent.com/camunda/camunda-platform-helm/main/charts/camunda-platform/values/values-v8.1.yaml
 ```
 
+### Connectors
+
+The **Connectors runtime** comes enabled by default. To start using ones, install Connector element
```

Comment:
The **Connectors runtime** comes enabled by default. To start using ones, install Connector element templates. Learn more in our documentation for [Web Modeler](/components/connectors/manage-connector-templates.md)
and [Desktop Modeler](/components/modeler/desktop-modeler/element-templates/configuring-templates.md).

Just need a little clarification here -- to start using "ones" -- what do you mean by this?

Are the links provided for Desktop Modeler and Web Modeler specific to installation? Instead of saying "Learn more in our documentation", I would note exactly what I user will find if they click on the link.

Reply:
How is this?

Connector runtime is enabled by default. To start using Connectors, install Connector element templates. Learn more about installing element templates in [Web Modeler](https://github.com/components/connectors/manage-connector-templates.md)
and [Desktop Modeler](https://github.com/components/modeler/desktop-modeler/element-templates/configuring-templates.md).

Reply:
@christinaausley looks good!